modules/stochastic_tools/doc/content/source/samplers/QuadratureSampler.md

```diff
@@ -0,0 +1,33 @@
+# QuadratureSampler
+
+!syntax description /Samplers/QuadratureSampler
+
+## Overview
+
+Numerical quadrature is a method of selecting specific points and weights in integrate a function,
+
+\begin{equation}
+\int_{a}^{b}y(x)f(x)dx = \sum_{q=1}^{N_q}w_qy(x_q) ,
+\end{equation}
```

Comment:
```suggestion
!equation
\int_{a}^{b}y(x)f(x)dx = \sum_{q=1}^{N_q}w_qy(x_q) ,

```

Reply:
I would like all the equations to use this syntax, the begin\end will be going away eventually.